Sources/CombineExternalAccessory/UnconfiguredExternalAccessory.swift

```diff
@@ -0,0 +1,131 @@
+//
+//  UnconfiguredExternalAccessory.swift
+//  WACTest
+//
+//  Created by Luis Reisewitz on 06.08.20.
+//  Copyright © 2020 Lautsprecher Teufel GmbH. All rights reserved.
+//
+
+import Combine
+import ExternalAccessory
+import Foundation
+import UIKit
+
+public struct UnconfiguredExternalAccessory {
+    /// Browser to use for configuring. We create a new one here, as we need to set the delegate.
+    /// Switching the delegate of the existing Browser used for discovering devices would hang the discovery
+    /// Publisher.
+    let browser = EAWiFiUnconfiguredAccessoryBrowser()
+    let accessory: EAWiFiUnconfiguredAccessory
+
+    // TODO: [COMBINE]: convert to promise later with foundationextension
+    /// - Warning: This completes the browser's search process and terminates the `Publisher`.
+    public func configure(on viewController: UIViewController) -> AnyPublisher<Void, ConfigurationError> {
+        return UnconfiguredExternalAccessoryPublisher(
+            browser: browser,
+            accessory: accessory,
+            viewController: viewController
+        ).eraseToAnyPublisher()
+    }
+
+    /// The reason why a configuration for a device can be unsuccessful.
+    public enum ConfigurationError: Error {
+        /// User cancelled the process.
+        case cancelled
+        /// The process failed in any way.
+        case failed
+    }
+}
+
+// MARK: - Equatable
+extension UnconfiguredExternalAccessory: Equatable {
+    public static func ==(lhs: Self, rhs: Self) -> Bool {
+        lhs.accessory == rhs.accessory
+    }
+}
+
+// MARK: - Hashable
+extension UnconfiguredExternalAccessory: Hashable {
+    public func hash(into hasher: inout Hasher) {
+        hasher.combine(accessory)
+    }
+}
+
+private struct UnconfiguredExternalAccessoryPublisher: Publisher {
+    typealias Output = Void
```

Comment:
If we want the Completable semantics, Output could be Never instead of Void, that way is impossible to send values, only completion (.finished or .failure)